pkgs/development/python-modules/terminaltables3/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltables3";
+  version = "4.0.0";
+  pyproject = true;
+
+  build-system = [
+    poetry-core
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tj7v4gmqiQBaCjTRUlc5QkVpcp7im15kqN1Rxevat38=";
+  };
+
+  pythonImportsCheck = [ "terminaltables3" ];
+
+  # doesn't have any tests
+  doCheck = false;
```

Comment:
Uhhhm, there are tests: https://github.com/matthewdeanmartin/terminaltables3/tree/master/tests

It can be that the tests aren't included on PyPi. But they should be when you change `src` to the GitHub-Repo.

Reply:
good idea, i'm gonna do that then.

Reply:
I don't think the fork cares about the tests

```
============================= test session starts ==============================
platform linux -- Python 3.12.7, pytest-8.3.3, pluggy-1.5.0
rootdir: /build/source
configfile: pyproject.toml
collected 801 items                                                            

tests/test_all_tables_e2e/test_ascii_table.py ..s                        [  0%]
tests/test_all_tables_e2e/test_double_table.py ..s                       [  0%]
tests/test_all_tables_e2e/test_github_table.py ..                        [  0%]
tests/test_all_tables_e2e/test_porcelain_table.py ..                     [  1%]
tests/test_all_tables_e2e/test_single_table.py ..                        [  1%]
tests/test_all_tables_e2e/test_single_table_windows.py sss               [  1%]
tests/test_ascii_table.py ......................                         [  4%]
tests/test_base_table/test_gen_row_lines.py ...............              [  6%]
tests/test_base_table/test_gen_table.py ....................             [  8%]
tests/test_base_table/test_horizontal_border.py ................         [ 10%]
tests/test_base_table/test_table.py .......F                             [ 11%]
tests/test_build/test_build_border.py .................................. [ 16%]
........................................................................ [ 25%]
........................................................................ [ 34%]
........................................................................ [ 43%]
........................................................................ [ 52%]
........................................................................ [ 61%]
...................................FFFFFFFFFFFFFFFFFF.FFFFFFFFFFFFFF.FFF [ 70%]
FFFFFFFFFFFFFFF.FFFFFFFFFFFFFF.....                                      [ 74%]
tests/test_build/test_build_row.py ......                                [ 75%]
tests/test_build/test_combine.py ......                                  [ 76%]
tests/test_build/test_flatten.py ..                                      [ 76%]
tests/test_examples.py ...                                               [ 76%]
tests/test_terminal_io/test_get_console_info.py .                        [ 76%]
tests/test_terminal_io/test_set_terminal_title.py sssssssss              [ 77%]
tests/test_terminal_io/test_terminal_size.py .....                       [ 78%]
tests/test_width_and_alignment/test_align_and_pad_cell.py ........F..... [ 80%]
......F...........F...........F...........F......F......F......F......F. [ 89%]
.....F................................F                                  [ 94%]
tests/test_width_and_alignment/test_column_max_width.py ...              [ 94%]
tests/test_width_and_alignment/test_max_dimensions.py ...........        [ 95%]
tests/test_width_and_alignment/test_table_width.py ...                   [ 96%]
tests/test_width_and_alignment/test_visible_width.py ................... [ 98%]
...........                                                              [100%]

=================================== FAILURES ===================================
__________________________________ test_color __________________________________
```

Reply:
A sample:
```
        if left and right:
            expected = left + expected + right
        actual = build_border(outer_widths, "-", left, intersect, right, title=title)
>       assert "".join(actual) == expected
E       AssertionError: assert 'TEST--------' == '\x1b[34mTEST\x1b[0m--------'
E         
E         - TEST--------
E         + TEST--------

```

Reply:
Very weird. I can replicate the failing tests. But I wonder why nearly every test got touched in the `4.0.0` commit https://github.com/matthewdeanmartin/terminaltables3/commit/f1c465b36eb9b91a984d8864b21376e7c37075b8

Reply:
looks like an autoformatting commit